dev-lisp/clisp_modules/clisp_modules-2.49.93~git.recipe

```diff
@@ -0,0 +1,101 @@
+SUMMARY="ANSI Common Lisp Implementation"
+DESCRIPTION="GNU CLISP is an ANSI Common Lisp Implementation. \
+It is known for high portability, small image size and efficient bignums. \
+\
```

Comment:
Backslashes not needed here, this way there will be an empty line in the Description.
```suggestion
It is known for high portability, small image size and efficient bignums.

```